tasks/__init__.py

```diff
@@ -258,25 +265,15 @@ def requirements(ctx, base=False, addons=False, release=False, dev=False, all=Fa
         addon_requirements(ctx)
     # "release" takes precedence
     if release:
-        req_file = os.path.join(HERE, 'requirements', 'release.txt')
-        ctx.run(
-            pip_install(req_file),
-            echo=True
-        )
+        install_from_txt(ctx, os.path.join(HERE, 'requirements', 'release.txt'))
     else:
         if dev:  # then dev requirements
-            req_file = os.path.join(HERE, 'requirements', 'dev.txt')
-            ctx.run(
-                pip_install(req_file),
-                echo=True
-            )
-
+            install_from_txt(ctx, os.path.join(HERE, 'requirements', 'dev.txt'))
         if base:  # then base requirements
-            req_file = os.path.join(HERE, 'requirements.txt')
-            ctx.run(
-                pip_install(req_file),
-                echo=True
-            )
+            install_from_txt(ctx, os.path.join(HERE, 'requirements.txt'))
+        if debug:  # then local debugging tools
+            install_from_txt(ctx, os.path.join(HERE, 'requirements', 'debug.txt'))
```

Comment:
Wondering about the orders of `dev`, `base` and `debug`. `debug` is another level after `dev` thus needs `dev`. I think updating the `-r` option in `debug.txt` to point `dev.txt` will solve this.